File: CSharp/Exercicio19.cs

public void SomaImpares(int numero)
{
    int j = 0;
    for(int i =0; i<= numero; i++)
    {
        if(i % 2 != 0)
        {
            Console.Write($"{i} + ");
            j += i;
        }
    }
    Console.Write($"= {j}");
}

SomaImpares(5)

File: JavaScript/laçosderepetição/Exercicio19.js

function somaImpares(numero) {
    let j = 0;
    for(i=1;i<=numero;i++) {
        if(i%2!=0){
            process.stdout.write(`${i} + `);
            j += i;
        }
    }
    process.stdout.write(`= ${j}`);
}

somaImpares(5)